src/transformers/models/bridgetower/image_processing_bridgetower.py

```diff
@@ -168,7 +172,7 @@ class BridgeTowerImageProcessor(BaseImageProcessor):
     def __init__(
         self,
         do_resize: bool = True,
-        size: Dict[str, int] = 288,
+        size: Dict[str, int] = {"shortest_edge": 288},
```

Comment:
This should be `None` here and set in the init - no mutables as default arguments